pybossa/view/projects.py

```diff
@@ -1610,3 +1642,19 @@ def webhook_handler(short_name, oid=None):
                            n_task_runs=n_task_runs,
                            n_completed_tasks=cached_projects.n_completed_tasks(project.get('id')),
                            n_volunteers=cached_projects.n_volunteers(project.get('id')))
+
+
+@blueprint.route('/<short_name>/results')
+def results(short_name):
+    """Results page for the project."""
+    (project, owner, n_tasks, n_task_runs,
+     overall_progress, last_activity,
+     n_results) = project_by_shortname(short_name)
+
+    title = project_title(project, "Results")
+
+    ensure_authorized_to('read', project)
```

Comment:
You're right. But then, we'll need to iterate over all results, and by definition results can be read by anyone, therefore should we check?